contrib/board-setup-helper.py

```diff
@@ -76,8 +77,9 @@ def get_device_numbers(serial_no, dev_type, board_file, ser2net_file):
                     for l in c.device_links:
                         if "by-id" in l:
                             port = ser2net(l, ser2net_file)
-                            if port:
+                            if (port and found_tty == False):
```

Comment:
A minor point: Python-native way to express this statement would be `if port and not found_tty:`

Reply:
Thanks for making my code more pythonic! Done.